lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/beans/SimplePageBean.java

```diff
@@ -5613,6 +5621,19 @@ public void maybeUpdateLinks() {
 
 	}
 
+	private boolean isCommentsItem(SimplePageItem item) {
+		return item.getType() == SimplePageItem.COMMENTS;
+	}
+
+	private int findNextSequence(List<SimplePageItem> items) {
+		int nseq = 0;
+		for (SimplePageItem item : items) {
+			if( item.getSequence() > nseq)
```

Comment:
please use braces or put on the same line.

Reply:
Made the change and created a new pull #6196 accordingly to sync with the JIRA. Maybe close this pull request?